src/test/harness.rs

```diff
@@ -219,11 +219,11 @@ pub fn expect_messages(results: LsResultList, expected: &[&ExpectedMessage]) {
 
     let mut results = results.lock().unwrap();
 
-    println!(
-        "expect_messages:\n  results: {:#?},\n  expected: {:#?}",
-        *results,
-        expected
-    );
+    // println!(
+    //     "expect_messages:\n  results: {:#?},\n  expected: {:#?}",
+    //     *results,
+    //     expected
+    // );
```

Comment:
Could this be restored?

Reply:
Oups, shouldn't have committed this, fixed!